public/media-hunt/general-data/normalizeddata.js

```diff
@@ -0,0 +1,50 @@
+//normalization of response
+
+const normalization = (data, actors) => ({
+  title: data.original_name || data.original_title,
+  year: data.first_air_date || data.release_date,
+  description: data.overview,
+  image: data.poster_path,
+  review: data.vote_average + " " + showStars(data.vote_average),
+  cast: actors,
+});
+
+//this is a response that gives us cast/credits
+const creditsResponse = {};
+
+//this is a logic that returs actors and their characters
+const actors = creditsResponse.cast.map((actorAndCharacter) => {
+  tvActors = {
+    name: actorAndCharacter.name,
+    character: actorAndCharacter.character,
+  };
+  return tvActors;
+});
+//this is a logic for stars calculation
+function halfStar(review) {
+  const smallStar = "⭐";
+  if ((review - Math.floor(review) >= 0.5)) return smallStar;
+  return {};
```

Comment:
```suggestion
  return "";
```

Reply:
Since we're working with strings, it won't make an error, but just to stay in the type

Reply:
done